pkg/storage/gcs/store.go

```diff
@@ -39,12 +41,30 @@ func clientOpts(readOnly bool, credentialFile string) []option.ClientOption {
 	return opts
 }
 
-// New builds a new storage object from a bucket string
-func New(ctx context.Context, bucket string, credentialFile string, opts ...Option) (storage.Store, error) {
+// versionedObject wraps *BucketHandle.Object
+// towards canonicalization of object names.
+func versionedObject(
```

Comment:
from #429 , canonicalization of object names is not anywhere on the horizon and `versionObject`'s body need be inlined